core/shared/src/main/scala/org/http4s/websocket/WebSocketFrameAggregator.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.websocket
+
+import fs2.Chunk
+import fs2.Pipe
+import scodec.bits.ByteVector
+
+private[http4s] object WebSocketFrameAggregator {
+  def aggregateFragment[F[_]]: Pipe[F, WebSocketFrame, WebSocketFrame] =
```

Comment:
I think "defragmentation" is more specific than "aggregation", what do you think? Naming stuff is hard.

```suggestion
private[http4s] object WebSocketFrameDefragmenter {
  def defragment[F[_]]: Pipe[F, WebSocketFrame, WebSocketFrame] =
```

Reply:
Unfortunately, I'm surprisingly bad at English😅 
Thanks for pointing it out!

[913768b](https://github.com/http4s/http4s/pull/7091/commits/913768b1f151a394625cb682c9310e5447f10835)